### PR TITLE
Add policy namespace to sanity label task workingDir

### PR DESCRIPTION
**Pull request by dirgim**
* Avoid overwriting task results by using different workingDir
* Add optional label check to hacbs-test-evaluation

### CODE REVIEW

**Review by staticf0x — APPROVED** [review]
LGTM

File: tasks/sanity-label-check.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ spec:
   steps:
     - name: basic-sanity-checks-required-labels
       image: quay.io/redhat-appstudio/hacbs-test:stable
-      workingDir: $(workspaces.workspace.path)/hacbs/$(context.task.name)
+      workingDir: $(workspaces.workspace.path)/hacbs/$(context.task.name)-$(inputs.params.POLICY_NAMESPACE)
```
Comment (Michkov) [inline tasks/sanity-label-check.yaml:22]: what is `inputs.params`? Cannot see it in https://tekton.dev/docs/pipelines/variables/#variables-available-in-a-task

Comment (dirgim) [reply tasks/sanity-label-check.yaml]: Weird, getting the parameter seems to work both with `params.POLICY_NAMESPACE` and `inputs.params.POLICY_NAMESPACE`.
Switched to `params` to be in line with the docs.

Comment (dirgim) [reply tasks/sanity-label-check.yaml]: Just figured it out, the inputs.params was used in v1alpha1 https://tekton.dev/docs/pipelines/migrating-v1alpha1-to-v1beta1/#changes-to-input-parameters

Comment (Michkov) [reply tasks/sanity-label-check.yaml]: I see that it's used multiple times already, changed from Alpha to beta -> https://tekton.dev/docs/pipelines/migrating-v1alpha1-to-v1beta1/#changes-to-fields , so we should probably update other occurrence in separate PR.

Comment (dirgim) [reply tasks/sanity-label-check.yaml]: Agreed, will raise it after this one.

**Review by Michkov — APPROVED** [review]

**Pull request merged**